src/components/PaginationFooter/Dropdowns/DropDown.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import type { SelectChangeEvent } from '@mui/material/Select';
+import { usePaginationContext } from '../context/PaginationProvider';
+import { StyledSelect, StyledTypography } from '../styles';
+
+interface Props {
+  name: string;
+  value: number;
+  options: JSX.Element;
+  handleChange: (event: SelectChangeEvent<number>) => void;
+}
+
+const DropDown = ({ name, value, options, handleChange }: Props) => {
+  const { layout, footerStyle, translator, tabIndex } = usePaginationContext();
+  const chartId = layout.qInfo.qId as string;
+  const label = translator.get(`NebulaTableUtils.Pagination.${name}`);
+  const baseId = `${name}-dropdown`;
```

Comment:
I forgot to add the `chartId` here in the first id fix PR. Have a look what I did here https://github.com/qlik-oss/sn-table/pull/1065/files#diff-24745d935b670fdf9cbdbe0ad22efcb2055f9bbe4fd19857939392f44e725f0bR127-R134

Reply:
Looks the same to me? :blind:

Reply:
you have already applied it, my bad!